### ISSUE TITLE
partial argument match of key to keys

### ISSUE THREAD

**Issue by LiNk-NY** [GeoBosh/Rdpack#28]
Hi @GeoBosh 

It looks like the `insertCite` hook is using a partial argument. 

I couldn't find it in the code but when testing a package with `\insertCite{...}`, I get the following warning: 

```r
Warning in Rdpack::insert_citeOnly(key = "wu2012camera", package = "zenith",  :
  partial argument match of 'key' to 'keys'
```

Best, 
Marcel

**Comment by GeoBosh** [comment]
Thanks for the report.
Could you give more details/context? Is this package `https://github.com/GabrielHoffman/zenith/`.?

**Comment by GeoBosh** [comment]
Indeed, the definitions of the Rdmacros in man/macros call insert_citeOnly` with `key` rather than `keys`. I will fix this. 

I don't get the warning though. I updated R-devel to the most recent one,  82650, and tried several packages which use this macro with no warnings when building/checking/installing. Do you get the warning when using `?xxx` in Rstudio or something similar?

**Comment by LiNk-NY** [comment]
Thank you Georgi, @GeoBosh 

I have these options in my `~/.Rprofile`:

```r
options(
    warnPartialMatchAttr = TRUE,
    warnPartialMatchDollar = TRUE,
    warnPartialMatchArgs = TRUE
)
```

**Comment by GeoBosh** [comment]
Very professional.:) Thank you Marcel, @LiNk-NY for reporting it.
I fixed this issue with commit [672591b](https://github.com/GeoBosh/Rdpack/commit/672591b2df5da3d30fbec0ac103421a13ae9b24e).

**Comment by LiNk-NY** [comment]
Great! Thank you for taking care of this so quickly!

Best regards,
Marcel
